Bio-Tech/Yael.md

```diff
@@ -0,0 +1,63 @@
+# Glossary
+## Biotechnology
+
+This glossary was created collaboratively with Our Best Words TechnComm students. This Biotechnology section was worked on by:
+[Yael Lachman](https://www.linkedin.com/in/yael-lachman/)
+[Rachel Goldstuck](https://www.linkedin.com/in/rachel-goldstuck/)
+[Eitan Gavson](https://www.linkedin.com/in/eitan-gavson/)
+
+[A](#a) [B](#b) C [D](#d) E F H G [I](#i) J K L [M](#m) N O [P](#p) Q [R](#r) S [T](#t) U V W X Y Z
+
+### A
+<dl>
+  <dt>autoimmune disease</dt>
+  <dd>a disease type in which the immune system can’t differentiate between its own cells and those which have invaded the body.</dd>
+</dl>
+
+### B
+<dl>
+  <dt>benign tumor</dt>
+  <dd>a collection of abnormal non-cancerous tissue cells that can grow uncontrollably and invade nearby tissues.</dd>
+</dl>
+
+### D
+<dl>
+  <dt>diigital health</dt>
+  <dd>the concept of providing medical treatments and information using technology.</dd>
+</dl> 
+
+### I
+<dl>
+  <dt>immunosuppressants</dt>
+  <dd>a type of medication that weaken the immune system, to prevent it causing harm to parts of the body that are needed, but that it may not recognize e.g. after an organ transplant.</dd>
+</dl>
+
+### M
+<dl>
+  <dt>metastasis</dt>
+  <dd>the process of a pathogen spreading from the initial area to a secondary area.</dd>
+  <dt>microbiology</dt>
+  <dd>the scientific study of all living organisms that are too small to be seen by a human’s eye.</dd>
+</dl>
+
+### P
+<dl>
+  <dt>pharmacogenomics</dt>
+  <dd>a field of research that studies how people’s genes affect their response to 
+</dd>
+</dl>
+
+### R
+<dl>
+  <dt>radiomics</dt>
+  <dd>a field of research that tries to extract large amounts of quantitative data from images by using data characterization algorithms.</dd>
```

Comment:
```suggestion
  <dd>A field of research applies data mining to medical imagery. This field focuses on pulling large amounts of quantitative data from images by using data characterization algorithms. By using data analysis and machine learning, applications can find tumors and other areas of concern faster and with better accuracy than humans. </dd>
```

Reply:
It seems to me that the **that** in the first sentence is needed, do you agree?